l10n_nl_tax_statement/tests/test_l10n_nl_vat_statement.py

```diff
@@ -403,7 +403,7 @@ def test_15_invoice_basis_undeclared_invoice(self):
             self.statement_1.company_id.country_id = self.env.ref('base.nl')
 
             invoice2 = self.invoice_1.copy()
-            old_date = fields.Date.today() + relativedelta(months=-4, day=1)
+            old_date = fields.Date.from_string('2018-12-07')
```

Comment:
imo this should be a separate FIX PR for the l10n_nl_tax_statement module.
@pedrobaeza what's your opinion on this?

Reply:
@pedrobaeza at this moment the branch 12.0 is broken because of this failing test. If I open a dedicated PR with this fix will you fast track it?